src/supermarket/.env

```diff
@@ -25,3 +25,4 @@ FIERI_FOODCRITIC_TAGS="metadata,correctness,supermarket,portability,deprecated ~
 FIERI_SUPERMARKET_ENDPOINT=http://localhost:13000
 ROBOTS_ALLOW=/
 ENFORCE_PRIVACY=true
+SPDX_LICENSE_URL=https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json
```

Comment:
We need to make sure that Supermarket can work with as little external connectivity as possible. Is it possible to fetch this file in the omnibus build so we ship Supermarket with the file.

Reply:
ok, will make this change

Reply:
Hi, How often do we expect this file to get updated? What will happen if the file gets updated since we have static copy of the same now?

Reply:
We should be pulling in the latest copy at build time in Supermarket.